server/src/main/kotlin/fi/oph/ludos/Validators.kt

```diff
@@ -79,7 +80,14 @@ annotation class SafeHtml(
     val groups: Array<KClass<*>> = [],
     val payload: Array<KClass<out Payload>> = []
 )
+
+val htmlSafelist: Safelist = Safelist.relaxed()
+    .addAttributes("img", "class")
```

Comment:
Aikasemmin oli sallittu class kaikille elementeille

```suggestion
    .addAttributes(":all", "class")
```